templates/extra/funding.html

```diff
@@ -0,0 +1,52 @@
+{% extends "base.html" %}
+
+{% block content %}
+<h2>How Much Should I Raise?</h2>
+<div style="display:flex; flex-direction: row; justify-content: center; align-items: center">
+    <form>
+        <h4>I want to raise ($)</h4>
+        <input type="text" id="raiseInput" placeholder="$1000" />
```

Comment:
Should the placeholder just be `1000`, since we don't want the user to enter the units?
```suggestion
        <input type="text" id="raiseInput" placeholder="1000" />
```